server/backend/api/app/crud/crud_application.py

```diff
@@ -92,6 +96,31 @@ def deleteFamApplication(db: Session, application_id: int):
     db.commit()
     return application
 
+
+def getFamApplicationRoles(
+        db: Session,
+        application_id: int) -> List[models.FamApplication]:
+    """Given a database session and an application id, will return a roles that
+    have been defined for the application id.  Currently does not return any
+    child roles.
+
+    :param db: input database session object
+    :param application_id: the application id who's roles are to be retrieved
+    :return: orm FamRole model listing related roles that have been created
+             for the given application.
+    """
+
+    application = (
+        db.query(models.FamRole)
+        # .join(models.FamRole)
+        # .options(load_only("application_id"))
+        .filter(models.FamRole.application_id == application_id,
+                models.FamRole.parent_role_id == None)
+        .all()
+    )
+    return application
```

Comment:
Hi Kevin, already noted in the meeting.
This probably should be a return of List[models.FamRole] type, and verify if the unit test also pass on this.